pandas/core/apply.py

```diff
@@ -302,6 +303,9 @@ def agg_list_like(self) -> DataFrame | Series:
         -------
         Result of aggregation.
         """
+        return self._apply_list_like(op_name="agg")
+
+    def _apply_list_like(self, op_name: Literal["agg", "apply"]) -> DataFrame | Series:
```

Comment:
Ok, I've renamed it. I didn't like `apply_list_like` because it infers it's similar to `_apply_list_like`, which it's not really, so I've named it `agg_or_apply_list_like`.

Reply:
Yea - that's a point of confusion for all of `core.apply`.